project/source_code/drivetrain/src/vehicle.cpp

```diff
@@ -7,14 +7,27 @@
 * @param receive_message_id id of the messages to read
 * @param transmit_message_id id of the messages to transmit. 0 if nothing to transmit
 */
-void Vehicle::Run() {
+bool Vehicle::Run() {
     //payload to be sent in canframe
     uint8_t payload[8] = {0,0,0,0,0,0,0,0};
+    uint32_t transmit_id = 2;
+    uint8_t transmit_length = 1;
```

Comment:
Could these also be defined as constants? Why does the transmit_id need to be a uint32?